content/docs/command-reference/config.md

```diff
@@ -238,6 +238,23 @@ to learn more about the state file (database) that is used for optimization.
   When an entry in the database is used (e.g. during the `dvc status`), DVC
   updates the timestamp on that entry. This way, when the database needs a
   cleanup, DVC can sort entries chronologically, and remove the oldest ones.
+  
+- `state.dir` - specify a custom location for the state file. This is
+  necessary when using DVC on NFS or other mounted volumes.
```

Comment:
> This is necessary when using DVC on NFS or other mounted volumes

@meierale is that always the case, or should it be "this may be necessary"?

Reply:
@jorgeorpinel That's a good question. To be honest I don't really know whether it is always necessary. It certainly is when working with azure ml. Maybe it's better to write "may be necessary", I agree.

Reply:
```suggestion
- `state.dir` - specify a custom location for the state file. This may be
  necessary when using DVC on NFS or other mounted volumes.
```